float32 robot_x
float32 robot_y
float32 robot_theta
int32 new_simulation
---
float32[1024] sensors
